pkg/providers/tinkerbell/validate.go

```diff
@@ -3,47 +3,16 @@ package tinkerbell
 import (
 	"errors"
 	"fmt"
-	"net/url"
 	"strings"
 
 	tinkv1alpha1 "github.com/tinkerbell/tink/pkg/apis/core/v1alpha1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
-	"github.com/aws/eks-anywhere/pkg/logger"
 	"github.com/aws/eks-anywhere/pkg/networkutils"
 	"github.com/aws/eks-anywhere/pkg/providers/tinkerbell/hardware"
 )
 
-func validateDatacenterConfig(config *v1alpha1.TinkerbellDatacenterConfig) error {
```

Comment:
I thought the providers team wanted to keep all validation in the `tinkerbell` package?
I'm totally ok with this, I have no concerns, just making sure everyone agrees it.

Reply:
Yes we discussed and agreed on this approach.